src/components/service-viewer/service-viewer.vue

```diff
@@ -138,11 +152,13 @@ export default {
       timeSpan: (state) => state.timeSpan,
       jobStatus: (state) => state.jobStatus,
       selectedTime: (state) => state.selectedTime,
-      statusLink: (state) => state.statusLink
+      statusLink: (state) => state.statusLink,
+      selectedCategory: (state) => state.selectedCategory,
```

Comment:
Why did you add them? They are not used in this component, so we can remove them? Or do you intend to use them?